### PR TITLE
Add FiscalCodeFake

### PR DESCRIPTION
**Pull request by francescopersico**

### CODE REVIEW

File: src/strings.ts [hunk 1]
```diff
@@ -137,6 +137,15 @@ export const FiscalCode = PatternString(
 
 export type FiscalCode = t.TypeOf<typeof FiscalCode>;
 
+/**
+ * A fake Fiscal Number (for persons)
```
Comment (cloudify) [inline src/strings.ts]:
```suggestion
 * A sandbox Fiscal Code (used for development)
 * Note that this pattern doesn't generate valid fiscal codes
```

File: src/strings.ts [hunk 2]
```diff
@@ -137,6 +137,15 @@ export const FiscalCode = PatternString(
...
+ */
+export const FiscalCodeFake = PatternString(
```
Comment (cloudify) [inline src/strings.ts]: I'd like to use a consistent naming for things that are "fake" and can be used for testing and development

```suggestion
export const SandboxFiscalCode = PatternString(
```

File: src/strings.ts [hunk 3]
```diff
@@ -137,6 +137,15 @@ export const FiscalCode = PatternString(
...
+  "^[A-Z]{6}[0-9]{2}A[0-9]{2}Y[0-9]{3}X$"
+);
+
+export type FiscalCodeFake = t.TypeOf<typeof FiscalCodeFake>;
```
Comment (cloudify) [inline src/strings.ts]:
```suggestion
export type SandboxFiscalCode = t.TypeOf<typeof SandboxFiscalCode>;
```

File: src/strings.ts (hunk 1 above)
Comment (cloudify) [reply src/strings.ts]: @gunzip @francescopersico could you check that the pattern doesn't indeed generate valid fiscal codes?

Comment (francescopersico) [reply src/strings.ts]:
```
/**
 * Generate a fake fiscal code.
 * Avoids collisions with real ones as we use
 * a literal "Y" for the location field.
 */
```

Comment (francescopersico) [reply src/strings.ts]: @cloudify The generated fiscalcode IS valid, is just associated to a (at the moment) not existing "Codice catastale"

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please include a Pivotal story at the beginning of the PR title (see below).</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>Please include a description of your PR changes.</td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cc6ec4bf822c5ff43c2e83f77f5442e61ce6e229
</p>

**Comment by cloudify** [comment]
published `v5.1.11`

**Pull request merged**